app/services/ubiquity/queue_import.rb

```diff
@@ -504,7 +504,7 @@ def self.upload(values)
             "contributor_position": "0"
         }
     ],
-    "id": "z2c4fd5e-a36d-461a-afb4-f91c9644de3p",
+    "id": "u2c4fd5e-a36d-561a-afb4-f91c9644de3t",
```

Comment:
What do we need this for?

Reply:
I use that for my local testing of the importer and it is in the commit  because when Rowan / Tom are in doubt of the right format or json keys to send to Hyku, they can just check the many examples of  json i used for testing.